### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'parse'

### ISSUE THREAD

**Issue by bsdz** [mvantellingen/python-zeep#237]
The below occurs on zeep 0.19.0 and in master. I can provide a stripped down wsdl but perhaps it something much simpler? Does the response look invalid?

```
zeep.transports: HTTP Post to http://service/url:
<?xml version='1.0' encoding='utf-8'?>
<soap-env:Envelope xmlns:ns1="http://foo.services.example.com/" xmlns:soap-env="http://schemas.xmlsoap.org/soap/envelope/">
  <soap-env:Body>
    <ns1:clearFoo>
      <code>scratch_code</code>
    </ns1:clearFoo>
  </soap-env:Body>
</soap-env:Envelope>

zeep.transports: HTTP Response from http://service/url (status: 200):
<?xml version="1.0" ?><S:Envelope xmlns:S="http://schemas.xmlsoap.org/soap/envelope/"><S:Body><ns2:clearFooResponse xmlns:ns2="http://foo.services.example.com/"></ns2:clearFooResponse></S:Body></S:Envelope>
E
======================================================================
ERROR: test_run (__main__.ClientTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "...../test_client.py", line 202, in test_run
    client.import_foo(foo_name, ....)
  File "...../client.py", line 145, in import_foo
    self.clearFoo(foo_name)
  File "...../site-packages/zeep-0.20.0-py2.7.egg/zeep/client.py", line 24, in __call__
    self._op_name, args, kwargs)
  File "...../site-packages/zeep-0.20.0-py2.7.egg/zeep/wsdl/bindings/soap.py", line 109, in send
    return self.process_reply(client, operation_obj, response)
  File "...../site-packages/zeep-0.20.0-py2.7.egg/zeep/wsdl/bindings/soap.py", line 147, in process_reply
    return operation.process_reply(doc)
  File "...../site-packages/zeep-0.20.0-py2.7.egg/zeep/wsdl/bindings/soap.py", line 300, in process_reply
    return self.output.deserialize(envelope)
  File "...../site-packages/zeep-0.20.0-py2.7.egg/zeep/wsdl/messages/soap.py", line 72, in deserialize
    body_result = self._deserialize_body(body)
  File "...../site-packages/zeep-0.20.0-py2.7.egg/zeep/wsdl/messages/soap.py", line 426, in _deserialize_body
    result = self.body.parse(response_element, self.wsdl.types)
AttributeError: 'NoneType' object has no attribute 'parse'

```

**Comment by bsdz** [comment]
Here's a unit test that reproduces error:

```python
def test_deserialize():
    wsdl_content = StringIO("""
    <definitions xmlns="http://schemas.xmlsoap.org/wsdl/"
                 xmlns:tns="http://tests.python-zeep.org/tns"
                 xmlns:soap="http://schemas.xmlsoap.org/wsdl/soap/"
                 xmlns:xsd="http://www.w3.org/2001/XMLSchema"
                 xmlns:wsam="http://www.w3.org/2007/05/addressing/metadata"
                 targetNamespace="http://tests.python-zeep.org/tns">

      
        <message name="clearFoo">
        <part name="code" type="xsd:string"/>
        </message> 
        
       <message name="clearFooResponse"/>      
      
      <portType name="TestPortType">
  
        <operation name="clearFoo">
            <input wsam:Action="http://foo.services.example.com/Util/clearFooRequest" message="tns:clearFoo"/>
            <output wsam:Action="http://foo.services.example.com/Util/clearFooResponse" message="tns:clearFooResponse"/>            
        </operation>     
        
      </portType>
      <binding name="TestBinding" type="tns:TestPortType">
        <soap:binding style="rpc" transport="http://schemas.xmlsoap.org/soap/http"/>

        <operation name="clearFoo">
            <soap:operation soapAction=""/>
            <input>
            <soap:body use="literal" namespace="http://foo.services.example.com/"/>
            </input>
            <output>
            <soap:body use="literal" namespace="http://foo.services.example.com/"/>
            </output>
        </operation>
        
        
      </binding>
    </definitions>
    """.strip())

    root = wsdl.Document(wsdl_content, None)

    binding = root.bindings['{http://tests.python-zeep.org/tns}TestBinding']
    operation = binding.get('clearFoo')

    document = load_xml("""
        <?xml version="1.0" ?>
        <S:Envelope xmlns:S="http://schemas.xmlsoap.org/soap/envelope/">
            <S:Body>
                <ns2:clearFooResponse xmlns:ns2="http://foo.services.example.com/"/>
            </S:Body>
        </S:Envelope>
    """)
    #assert operation.output.signature(True) == 'body: {result: xsd:string}, header: {}'
    result = operation.output.deserialize(document)
    assert result == ''
```

**Comment by mvantellingen** [comment]
Might be fixed in 0.20.0. Was a bit of a quick release due to some stuff I required for one of my own projects.

**Comment by bsdz** [comment]
Your last commits appear to have resolved it - thanks!
